[2024-04-23 03:01:04,893] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos2.cria_pasta scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-23 03:01:04,898] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dados_climaticos2.cria_pasta scheduled__2024-04-15T00:00:00+00:00 [queued]>
[2024-04-23 03:01:04,898] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-04-23 03:01:04,898] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2024-04-23 03:01:04,898] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-04-23 03:01:04,909] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): cria_pasta> on 2024-04-15 00:00:00+00:00
[2024-04-23 03:01:04,911] {standard_task_runner.py:52} INFO - Started process 150795 to run task
[2024-04-23 03:01:04,913] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'dados_climaticos2', 'cria_pasta', 'scheduled__2024-04-15T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dados_climaticos.py', '--cfg-path', '/tmp/tmpu57g00nv', '--error-file', '/tmp/tmp4goahk8p']
[2024-04-23 03:01:04,914] {standard_task_runner.py:80} INFO - Job 16: Subtask cria_pasta
[2024-04-23 03:01:04,941] {task_command.py:370} INFO - Running <TaskInstance: dados_climaticos2.cria_pasta scheduled__2024-04-15T00:00:00+00:00 [running]> on host URATIC9519.RIOBRANCOPETROLEO
[2024-04-23 03:01:04,973] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dados_climaticos2
AIRFLOW_CTX_TASK_ID=cria_pasta
AIRFLOW_CTX_EXECUTION_DATE=2024-04-15T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-04-15T00:00:00+00:00
[2024-04-23 03:01:04,974] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2024-04-23 03:01:04,974] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'mkdir -p "/home/enrico/Documentos/backup/AULA/Alura/Engenharia de dados/projeto_airflow/airflowalura/semana=2024-04-22"']
[2024-04-23 03:01:04,980] {subprocess.py:85} INFO - Output:
[2024-04-23 03:01:04,981] {subprocess.py:96} INFO - Command exited with return code 0
[2024-04-23 03:01:04,994] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=dados_climaticos2, task_id=cria_pasta, execution_date=20240415T000000, start_date=20240423T060104, end_date=20240423T060104
[2024-04-23 03:01:05,005] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-04-23 03:01:05,023] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
